en/python_installation/instructions.md

```diff
@@ -17,6 +17,10 @@ In upcoming steps, you'll be using the Windows Command Line (which we'll also te
 
 ![Type "cmd" in the "Run" window](../python_installation/images/windows-plus-r.png)
 
+Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python 3.5.x installer fails with an error, you can try either:
+1. install all Windows Updates and try to install Python 3.5 again; or
+2. install an older version of Python, e.g., 3.4.4.
```

Comment:
My comment was not clear :sweat_smile: : can you create a link on "older version of Python" pointing to that page. Thanks :)